[2021-06-27 07:23:49,822] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_postgresql_invoice 2021-06-27T07:23:39.796148+00:00 [queued]>
[2021-06-27 07:23:49,867] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_postgresql_invoice 2021-06-27T07:23:39.796148+00:00 [queued]>
[2021-06-27 07:23:49,868] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 07:23:49,869] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-06-27 07:23:49,872] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 07:23:49,891] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): export_postgresql_invoice> on 2021-06-27T07:23:39.796148+00:00
[2021-06-27 07:23:49,901] {standard_task_runner.py:52} INFO - Started process 5476 to run task
[2021-06-27 07:23:49,909] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgretomysql', 'export_postgresql_invoice', '2021-06-27T07:23:39.796148+00:00', '--job-id', '52', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/postgresql-collect.py', '--cfg-path', '/tmp/tmp8fnusnij', '--error-file', '/tmp/tmpwx8sb22c']
[2021-06-27 07:23:49,912] {standard_task_runner.py:77} INFO - Job 52: Subtask export_postgresql_invoice
[2021-06-27 07:23:49,990] {logging_mixin.py:104} INFO - Running <TaskInstance: postgretomysql.export_postgresql_invoice 2021-06-27T07:23:39.796148+00:00 [running]> on host b69deda3a989
[2021-06-27 07:23:50,075] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=tech@innospark.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgretomysql
AIRFLOW_CTX_TASK_ID=export_postgresql_invoice
AIRFLOW_CTX_EXECUTION_DATE=2021-06-27T07:23:39.796148+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-27T07:23:39.796148+00:00
[2021-06-27 07:23:50,076] {logging_mixin.py:104} INFO - Entering export_postgresql_to_csv SELECT * FROM test.invoice
[2021-06-27 07:23:50,089] {base.py:78} INFO - Using connection to: id: docker-postgres. Host: 192.168.1.27, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-06-27 07:23:51,545] {base.py:78} INFO - Using connection to: id: docker-postgres. Host: 192.168.1.27, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-06-27 07:23:51,565] {python.py:151} INFO - Done. Returned value was: None
[2021-06-27 07:23:51,590] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=postgretomysql, task_id=export_postgresql_invoice, execution_date=20210627T072339, start_date=20210627T072349, end_date=20210627T072351
[2021-06-27 07:23:51,670] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-27 07:23:51,769] {local_task_job.py:151} INFO - Task exited with return code 0
